### ISSUE TITLE
some ui pieces missing from tab order

### ISSUE THREAD

**Issue by dericed** [MediaArea/BWFMetaEdit#279]
I'm reviewing 
- Keyboard users cannot navigate to or activate any of the interactive elements on the Files screen, including Edit/Save/Remove, or any of the pop-out controls.
- Ensure that all interactive elements receive keyboard focus and can be activated without using a mouse.

In the latest, I can tab between the boxes in this image
<img width="673" alt="image" src="https://github.com/MediaArea/BWFMetaEdit/assets/762604/230ff3b8-2d7d-4622-adc8-d63ed7035350">

but I can't tab to select the arrow button that opens the editor, thus in this layout I need the mouse to edit.
